compose/service.py

```diff
@@ -540,6 +540,10 @@ def _next_container_number(self, one_off=False):
 
     def _get_links(self, link_to_self):
         links = []
+
+        if self.use_networking:
```

Comment:
Minor: Could you put this line before the `links = []` line. It helps express the intent a little clearer that if we're using networking that `_get_links` is to do nothing.